internal/pkg/cli/validate.go

```diff
@@ -280,17 +283,51 @@ func validatePath(fs afero.Fs, val interface{}) error {
 	return nil
 }
 
-func validateStorageType(val interface{}) error {
+type validateStorageTypeOpts struct {
+	ws           manifestReader
+	workloadName string
+}
+
+func validateStorageType(val interface{}, opts validateStorageTypeOpts) error {
 	storageType, ok := val.(string)
 	if !ok {
 		return errValueNotAString
 	}
-	for _, validType := range storageTypes {
-		if storageType == validType {
-			return nil
-		}
+	if !contains(storageType, storageTypes) {
+		return fmt.Errorf(fmtErrInvalidStorageType, storageType, prettify(storageTypes))
+	}
+
+	if storageType == rdsStorageType {
+		return validateAuroraStorageType(opts.ws, opts.workloadName)
+	}
+	return nil
+}
+
+func validateAuroraStorageType(ws manifestReader, workloadName string) error {
+	if workloadName == "" {
```

Comment:
Should we validate again after getting the workload name?

Reply:
We do! `validateStorageType` is called both in `Validate` and `Ask` :)